### ISSUE TITLE
translate hello 

### ISSUE THREAD

**Issue by Tevychhy** [Tevychhy/tevy.github.io#1]
Hi guys, so this is easy and fun for first timer.
All you need to do is display hello in using different language put it in the div text and put different color.
Highly recommend for first-timer who only know a basic about HTML and CSS.
![screenshot 87](https://user-images.githubusercontent.com/45750360/53399609-e3ddd600-39de-11e9-8a59-782001b7b02c.png)

**Comment by cmatiello** [comment]
Hi, @Tevychhy! 
Can I do this? I'm learning HTML, CSS and how to contribute to open source =)

**Comment by Tevychhy** [comment]
> Hi, @Tevychhy!
> Can I do this? I'm learning HTML, CSS and how to contribute to open source =)

**Comment by Tevychhy** [comment]
@cmatiello  yes sure, You should try this out. Translate to your language . Good luck.
